README.md

```diff
@@ -21,6 +21,26 @@ When you run `npm run build` the terminal will output the error, including the h
 
 ![CodeHighlight](https://cloud.githubusercontent.com/assets/175278/22310149/1ee66ccc-e346-11e6-83ff-e3a053701fb4.gif)
 
+## Migration
+
+In general, most upgrades wont require any migration steps to work, but if you experience problems after an upgrade, please file an issue, and we'll add it to the list of migration steps below.
+
+### From `<2.5.0` to `>=2.5.0`
+
+Version `2.5.0` introduces a new config file for jest, that is necessary for the tests to run. If you were previously running a version older than `v2.5.0` and upgraded to `v2.5.0` or newer, you need to manually add the new file, or else you'll get an error similar to this when trying to run your tests:
+
+```javascript
+Test suite failed to run
+
+{
+    "messageText": "Cannot read file 'C:\\[project]\\tsconfig.test.json': ENOENT: no such file or directory, open 'C:\\[project]\\tsconfig.test.json'.",
+    "category": 1,
+    "code": 5012
+}
+```
+
+To fix this, create a new file *in the root of the project* called `tsconfig.test.json`, and paste [the content of this file into it](https://raw.githubusercontent.com/wmonk/create-react-app-typescript/master/packages/react-scripts/template/tsconfig.test.json). Everything should work now. For more info, please see [this issue](https://github.com/wmonk/create-react-app-typescript/issues/141).
```

Comment:
Should we link to a commit version of `tsconfig.test.json` instead of the master version? Could cause issues in the future if the file doesn't exist.

Reply:
I thought about that to, however I think that linking to the master version is more future proof. Say we release a version eg. `2.11.0` that requires some additional configurations in that file, when one tries to migrate from `2.4.0` to `2.11.0` they would have to insert the new configs as well, and not just the ones introduced in `2.5.0`. If we one day remove the config all together, maybe we should change the migration guide to say something like "From `<2.5.0` to `2.5.0 - 2.10.0`" to emphasise that the configuration is only necessary between those version, and was removed later on.

Thoughts?

Reply:
Yeah sounds good to me. Lets leave as is!